Makefile

```
@@ -24,9 +24,12 @@ clean:
 	rm -f *deb *rpm
 	rm -f sha256sum md5sum
 
-$(NAME): $(wildcard **/*.go)
+$(NAME):
 	$(GO) build $(MODULE)
 
+e2e:
```

```suggestion
e2e: $(NAME)
```

It looks like it should depend on the main binary